src/applications/simple-forms/form-upload/containers/ReviewPage.jsx

```diff
@@ -52,9 +75,12 @@ const ReviewPage = () => {
             {capitalize(fullName.first)} {capitalize(fullName.last)}
           </b>
         </p>
-        <p>Social Security number: TODO: Insert redacted SSN</p>
-        <p>VA file number: TODO: Insert redacted File Number</p>
-        <p>Zip code: TODO: Insert zip code</p>
+        {veteran && (
+          <>
+            <p>Social Security number: {veteran.ssn}</p>
```

Comment:
Changes look good for a frontend solution! I think SSN needs to be masked if we are showing it on the screen. There's an example in line 19 of appeals/shared/components/VeteranInformation.jsx. where it masks, but screen reader still reads out nicely "ending with...."

Reply:
Good call! I've updated the branch to use this masking logic. Is it a problem that the FE is still fetching the full SSN?
<img width="785" alt="Screenshot 2024-05-21 at 9 10 40 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/10481391/a0a3a323-5662-43df-95e0-7c99f460d722">